File: Problem 05 (updated).py
# Get input from the user
a = 5
b = 7
c = 3

# Use nested if conditions to find the greatest number
if a > b:
  if a > c:
    greatest = a
  else:
    greatest = c
else:
  if b > c:
    greatest = b
  else:
    greatest = c

# Print the greatest number
print("The greatest number is:", greatest)

File: Problem 05 (updated).r

a = 5
b = 3
c = 9

if (a > b) {
   if (a > c) {
      greatest = a
   } else {
      greatest = c
   }
} else {
   if (b > c) {
      greatest = b
   } else {
      greatest = c
   }
}


cat("The greatest number is:", greatest, "\n")
